plugins/modules/openshift_adm_prune_builds.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2021, Red Hat
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+
+module: openshift_adm_prune_builds
+
+short_description: Prune old completed and failed builds
+
+version_added: "2.2.0"
+
+author:
+  - Aubin Bikouo (@abikouo)
+
+description:
+  - This module allow administrators to delete old completed and failed builds.
+  - Analogous to C(oc adm prune builds).
+
+extends_documentation_fragment:
+  - kubernetes.core.k8s_auth_options
+
+options:
+  namespace:
+    description:
+    - Use to specify namespace for builds to be deleted.
+    type: str
+  keep_younger_than:
+    description:
+    - Specify the minimum age (in minutes) of a Build for it to be considered a candidate for pruning.
+    type: int
+  orphans:
+    description:
+    - If C(true), prune all builds whose associated BuildConfig no longer exists and whose status is
+      complete, failed, error, or cancelled.
+    type: bool
+    default: False
+
+requirements:
+  - python >= 3.6
+  - kubernetes >= 12.0.0
+'''
+
+EXAMPLES = r'''
+# Run deleting older completed and failed builds and also including
+# all builds whose associated BuildConfig no longer exists
+- name: Run delete orphan Builds
+  openshift_adm_prune_builds:
```

Comment:
I guess we also would need to add the FQDN.

Reply:
yes sure